twine/commands/check.py

```diff
@@ -94,7 +94,7 @@ def check(dists, output_stream=sys.stdout):
 
         # Actually render the given value
         rendered = renderer.render(
-            metadata.get("description"), stream=stream, **parameters
+            metadata.get("description") or "", stream=stream, **parameters
```

Comment:
It would be better if this was done outside of the function call.

It would further seem as though `"description"` may not be present and if it is present (or perhaps it is always present?) it can be `None`. The behaviour of `metadata` seems to be poorly understood by all involved, what do you think about documenting the actual behaviour here. It would be better if everyone after this point in time understood the reason for this change and why it was necessary. The comment in the test is a bit too opaque to provide that understanding.

Reply:
I was following the precedent from line 89, would you like me to change that one as well?

Reply:
Please do. It's not a very good precedent.